Esp8266/Esp8266_Interface.pb.c

```diff
@@ -0,0 +1,27 @@
+/* Automatically generated nanopb constant definitions */
```

Comment:
So its usually not good practice to include generated files along with the repository, the ideal way to do it would be to add detailed steps on how to get set up and generate the files from the proto so if the end user wants to change the proto they can, very easily.

Reply:
Yes boss. This seems like a job for Mr. gitignore over in root.